### PR TITLE
feat : completed hoome page

### PR DESCRIPTION
**Pull request by shaikhhamza11**
1)Created  product context and api request for product and categories using hooks

### CODE REVIEW

**Review by Richa-rani02 — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -20,39 +20,16 @@
...
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
     <title>SportsKart</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root">Loading...</div>
```
Comment (sharathnair9999) [inline public/index.html:33]: why is this loading... for?

**Review by ajay-mandaviya — APPROVED** [review]

File: src/components/Hero/Hero.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  background: var(--bg-primary);
+  padding: 0 8rem;
+  position: relative;
+  z-index: 5;
+}
+.Hero .hero-img {
+  grid-area: hero-img;
+  display: flex;
+  align-items: center;
+  width: 50rem;
```
Comment (sharathnair9999) [inline src/components/Hero/Hero.css:16]: you can try max-width so it can be responsive

File: src/components/ProductCard/ProductCard.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
+.product {
+  width: 23%;
```
Comment (sharathnair9999) [inline src/components/ProductCard/ProductCard.css:2]: try to use absolute width for a product card.

**Review by sharathnair9999 — APPROVED** [review]
Looks good. You can reduce number of files in commit from next time.

File: public/index.html (hunk 1 above)
Comment (shaikhhamza11) [reply public/index.html:33]: It will show any text we put between before our apps load

File: src/components/Hero/Hero.css (hunk 2 above)
Comment (shaikhhamza11) [reply src/components/Hero/Hero.css:16]: will fix in the next pr

File: src/components/ProductCard/ProductCard.css (hunk 3 above)
Comment (shaikhhamza11) [reply src/components/ProductCard/ProductCard.css:2]: will fix in the next pr

**Pull request merged**